stories/module-analytics-settings.stories.js

```diff
@@ -342,6 +348,113 @@ storiesOf( 'Analytics Module/Settings', module )
 			decorators: [ WithRegistry ],
 		}
 	)
+	.add(
+		'Edit, open with all settings w/ GA4, w/o module access',
+		( args, { registry } ) => {
+			const { dispatch } = registry;
+			const {
+				accounts,
+				properties,
+				profiles,
+			} = accountsPropertiesProfiles;
+
+			/* eslint-disable sitekit/acronym-case */
+			const {
+				accountId: accountID,
+				webPropertyId,
+				id: profileID,
+			} = profiles[ 0 ];
+			const { internalWebPropertyId } = properties.find(
+				( property ) => webPropertyId === property.id
+			);
+			/* eslint-enable */
+
+			provideModules( registry, [
+				{
+					slug: 'search-console',
+					active: false,
+					connected: true,
+				},
```

Comment:
We should provide the owner id for these too.

Reply:
So the `provideModules`'s `owner.id` does not take effect in these stories and I have to add `ownerID: 1` within each `receiveGetSettings` call - I think the settings is overwriting the owner. But again, we can get away without making any changes here due [to us checking if `hasModuleAccess === false` instead of  `! hasModuleAccess` ](https://github.com/google/site-kit-wp/pull/5301#discussion_r887790859) in all `*Select` components.